CONTRIBUTE.md

```diff
@@ -0,0 +1,102 @@
+# Contributing Guidelines for firecamp.io
+
+Thank you for considering contributing to firecamp.io! To ensure smooth collaboration and maintain a consistent workflow, please follow the guidelines below.
+
+## Commit Guidelines
+
+- We encourage using either the learna or commitzen format for your commit messages.
```

Comment:
should we add the ref links for learna or commitzen?

Reply:
`conventional commits` link 👍